src/redux/sliceResources/resources.ts

```diff
@@ -0,0 +1,72 @@
+import { createUrlPath } from '~/utils/helper-functions'
+import { URLs } from '~/constants/request'
+import { appApi } from '~/redux/apiSlice'
+
+import { Attachment, ItemsWithCount, Lesson, LessonData } from '~/types'
+
+export interface LessonsQueryArgs {
+  limit: number
+  skip: number
+  sort: object
+  title: string
+  categories: string[]
+}
+type DeleteLessonArgs = string
+
+interface EditLessonArgs {
+  id?: string
+  title: string
+  description: string
```

Comment:
move interfaces and types to types folder

Reply:
fixed